### PR TITLE
Add feature booking event

### PR DESCRIPTION
**Pull request by yongmin86k**
# Add feature booking event

- Add the function on onPress button at BookingEvent screen with async-storage method.

### CODE REVIEW

File: client/js/screens/EventBooking/EventBookingContainer.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          <StatusBar
+            barStyle={themeColor === 'light' ? 'dark-content' : 'light-content'}
+          />
+        );
+      },
+
+      headerLeft: () => {
+        return (
+          <TouchableOpacity
+            style={{marginLeft: 12}}
```
Comment (siddharthparmar7) [inline client/js/screens/EventBooking/EventBookingContainer.js]: Move this inline style from here to styles.js file

File: client/js/screens/EventBooking/EventBookingContainer.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                  ? require('../../assets/images/icArrLeftDefault.png')
+                  : require('../../assets/images/icArrLeftWhite.png')
+              }
+            />
+          </TouchableOpacity>
+        );
+      },
+      headerRight: () => (
+        <TouchableOpacity
+          style={{marginRight: 12}}
```
Comment (siddharthparmar7) [inline client/js/screens/EventBooking/EventBookingContainer.js]: same as above.

File: client/js/screens/ThankYou/ThankYouContainer.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        backgroundColor: 'transparent',
+      },
+
+      headerBackground: () => {
+        return <StatusBar barStyle="dark-content" />;
+      },
+
+      headerRight: () => (
+        <TouchableOpacity
+          style={{marginRight: 12}}
```
Comment (siddharthparmar7) [inline client/js/screens/ThankYou/ThankYouContainer.js]: same as above.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes.

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts.

File: client/js/screens/EventBooking/EventBookingContainer.js (hunk 1 above)
Comment (yongmin86k) [reply client/js/screens/EventBooking/EventBookingContainer.js]: Changed

File: client/js/screens/EventBooking/EventBookingContainer.js (hunk 2 above)
Comment (yongmin86k) [reply client/js/screens/EventBooking/EventBookingContainer.js]: Changed

File: client/js/screens/ThankYou/ThankYouContainer.js (hunk 3 above)
Comment (yongmin86k) [reply client/js/screens/ThankYou/ThankYouContainer.js]: Changed

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**